core/api/v1/keytransparency.proto

```diff
@@ -313,15 +313,15 @@ service KeyTransparency {
   // Clients verify their account history by observing correct values for their
   // account over time.
   rpc ListEntryHistory(ListEntryHistoryRequest) returns (ListEntryHistoryResponse) {
-    option (google.api.http) = { get: "/v1/directories/{directory_id}/users/{user_id}/history" };
+    option (google.api.http) = { get: "/v1/history/directories/{directory_id}/users/{user_id}" };
   }
 
   // QueueUserUpdate enqueues an update to a user's profile.
   //
   // Clients should poll GetEntry until the update appears, and retry if no update appears after a timeout.
   rpc QueueEntryUpdate(UpdateEntryRequest) returns (google.protobuf.Empty) {
     option (google.api.http) = {
-      post: "/v1/directories/{directory_id}/users/{user_id}:queue"
+      post: "/v1/directories/{directory_id}/users/{user_id}"
```

Comment:
The `post` doesn't really look like one of the standard methods in 
https://cloud.google.com/apis/design/standard_methods, which is why it had the custom verb per
https://cloud.google.com/apis/design/custom_methods. What do you think?